_pkgdown.yml

```diff
@@ -22,7 +22,6 @@ reference:
   - tf::tf_approx_linear
   - tf::tf_evaluate
   - tf::tf_interpolate
-  - tf::tfsplitcombine
```

Comment:
why did you remove this?

Reply:
it's not defined anywhere?

Reply:
should be `tf::tf_splitcombine`, you're right